modules/loadbalancer/templates/nginx_balance.conf.erb

```diff
@@ -89,6 +89,12 @@ server {
     allow 10.0.0.0/8;
     allow 127.0.0.1;
     deny all;
+    <%- elsif @aws_egress_nat_ips -%>
+    # Only accept connnections from AWS Environment
+    <%- [@aws_egress_nat_ips].flatten.each do |ip| -%>
+    allow <%= ip %>;
```

Comment:
I will put `deny all` statement in line 91 and 97 after line 98. Should be there in all cases

Reply:
not sure if I follow this... if there is a "deny all" after line 98 then nothing can use the that nginx load balancer unless it has @internal_only or @aws_egress_nat_ips defined... there are nginx configs that allow from everywhere by not having any allow or deny in the config

Reply:
cool, I get it.

Reply:
cool, got it.